ResSimpy/Nexus/NexusSimulator.py

```
@@ -81,7 +81,7 @@ def __init__(self, origin: Optional[str] = None, destination: Optional[str] = No
             ValueError: If the FCS file path is not given
 
         Examples:
-            Example of calling a NexusSimulator with a file path to an fcs file:
+            Example of calling a NexusSimulator with a file path to a fcs file:
```

I don't agree with this change as the sound of fcs is a vowel so should be "an fcs"

---

agreed, it was showing up as an error for some reason, so I changed it back to "an fcs" and ignored the error.